ci/ansible/roles/pulp-certs/tasks/main.yml

```diff
@@ -1,22 +1,32 @@
 ---
 - name: Create CA to sign all certificates
   command: >
-    /usr/bin/openssl req -x509 -newkey rsa:2048 -keyout private/cakey.pem -nodes -days 3650
-     -out cacert.pem -subj '/C=US/ST=North Carolina/L=Raleigh/O=Pulp/OU=Development/CN=PulpCA'
+    /usr/bin/openssl req -x509 -newkey rsa:2048 -keyout private/cakey.pem -nodes -days 3650 -out cacert.pem -subj '/C=US/ST=North Carolina/L=Raleigh/O=Pulp/OU=Development/CN=PulpCA'
```

Comment:
I missed this, have joined the lines to test manually on terminal. I will revert it.